src/components/Layouts/Collection.tsx

```diff
@@ -1,30 +1,42 @@
-import { ActionTypes, useAppContext } from "../../context/AppContext";
 import {
   Box,
-  Button,
-  Container,
   Em,
-  Flex,
-  Heading,
   Section,
   TableBody,
   TableColumnHeaderCell,
   TableHeader,
   TableRow,
   Text,
 } from "@radix-ui/themes";
-import React, { MouseEventHandler } from "react";
-import { data, projectTitle } from "../../data";
+import React, { useEffect, useState } from "react";
 
+import { ManifestEditorManifest } from "../../types/manifest-editor";
 import UITable from "../UI/Table/Table";
 import UITableCollectionItemsRow from "../UI/Table/CollectionItemsRow";
+import { projectTitle } from "../../data";
+import { useAppContext } from "../../context/AppContext";
 
 const Collection = () => {
-  const { dispatch } = useAppContext();
+  const [manifests, setManifests] = useState<ManifestEditorManifest[]>([]);
 
-  const handleLogout: MouseEventHandler<HTMLButtonElement> = () => {
-    dispatch({ type: ActionTypes.LOGOUT });
-  };
+  const { state } = useAppContext();
+  const { authToken } = state;
+
+  useEffect(() => {
+    const baseUrl =
+      "https://67qdcv50f4.execute-api.us-east-1.amazonaws.com/prod";
+
+    fetch(`${baseUrl}/manifests`, {
```

Comment:
For eventual tests, you'll likely want to move this network request to an external function, ie. `fetchManifests()`, which can be mocked.